### PR TITLE
feat(securityLog): add support

### PR DESCRIPTION
**Pull request by JacopoDaeli**
Adding support for optional security log function. This PR is for the old warehouse @6.

### CODE REVIEW

**Review by jcrugzz — APPROVED** [review]
Seems fine. We might want to add an assert somewhere where we do auth that it gets called thooug.

File: lib/middlewares/index.js [hunk 1]
```diff
@@ -4,13 +4,40 @@ const cors = require('access-control');
 const debugParam = require('./debug');
 
 module.exports = function (app, options, done) {
+  var authMiddleware = app.authboot && app.authboot.middleware || function (req, res, next) { next(); };
```
Comment (decompil3d) [inline lib/middlewares/index.js:7]:
```suggestion
  const authMiddleware = app.authboot && app.authboot.middleware || function (req, res, next) { next(); };
```

**Review by decompil3d — CHANGES_REQUESTED** [review]
We also need to log for writes -- is that coming in a later PR?

**Comment by JacopoDaeli** [comment]
@decompil3d  yes, there is going to be another PR for the writes.

**Review by decompil3d — APPROVED** [review]

**Pull request merged**